python/lsst/dax/imgserv/jobqueue/imageworker.py

```diff
@@ -78,17 +80,30 @@ def get_image_async(self, *args, **kwargs):
     config["DAX_IMG_CONFIG"] = config_path
     meta_url = imgserv_config.webserv_config["dax.imgserv.meta.url"]
     config["DAX_IMG_META_URL"] = meta_url
-    soda = imageSODA.ImageSODA(config)
-    image = soda.do_sync(params)
-    # save the result image in local temp dir
-    with tempfile.NamedTemporaryFile(dir=config["DAX_IMG_TEMPDIR"],
+    image = None; status = "OK"
+    try:
+        soda = imageSODA.ImageSODA(config)
+        image = soda.do_sync(params)
+    except ImageNotFound:
+        status = "Error=Image Not Found"
+    except UsageError as e:
+        status = "Error="+e.message
```

Comment:
If you change the exceptions to be standard Python 3 style then this would be:
```python
status = f"Error={e}"
```
and in some sense this could be the same for any exception since this is effectively what ImageNotFound is doing and probably normal Exception.

Reply:
I took this as a lead to simplify the new exceptions.